scripts/lib/install

```diff
@@ -325,117 +323,101 @@ fi
 
 # Generate /etc/zulip/zulip.conf .
 mkdir -p /etc/zulip
-if [ -z "$NO_OVERWRITE_SETTINGS" ] || ! [ -e "/etc/zulip/zulip.conf" ]; then
-    (
-        cat <<EOF
+has_class() {
+    grep -qx "$1" /var/lib/puppet/classes.txt
+}
+
+# puppet apply --noop fails unless the user that it _would_ chown
+# files to exists; https://tickets.puppetlabs.com/browse/PUP-3907
+useradd -m zulip
```

Comment:
I think we probably want either `-M` or `--home-dir /home/zulip`; otherwise if the sysadmin has configured homedirs in some unusual way this could result in issues.